import.py

```diff
@@ -314,7 +313,7 @@ def _report_environment():
     force_existing_branch = True if os.environ.get('CF_IMPORT_FORCE_EXISTING_BRANCH', 'false').strip().lower() == "true" else False  # noqa: E501
     protect_branch = True if os.environ.get('CF_IMPORT_PROTECT_BRANCH', 'true').strip().lower() == "true" else False  # noqa: E501
     repo_privacy = True if os.environ.get('CF_IMPORT_PRIVATE_REPO', 'true').strip().lower() == "true" else False  # noqa: E501
-    gitea_url = urljoin(gitea_base_url, '/api/v1')
+    gitea_url = f'{gitea_base_url.rstrip("/")}/api/v1'
```

Comment:
This is fine as is, but for future reference switching to os.path.join probably would have been cleaner.